plugins/poll/assets/javascripts/discourse/components/poll-option-irv.gjs

```diff
@@ -0,0 +1,40 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import routeAction from "discourse/helpers/route-action";
+import i18n from "discourse-common/helpers/i18n";
+import PollOptionIrvDropdown from "./poll-option-irv-dropdown";
+
+export default class PollOptionsComponent extends Component {
+  @service currentUser;
+
+  @action
+  sendRank(option, rank = 0) {
+    this.args.sendRank(option, rank);
+  }
+
+  <template>
+    <div
+      tabindex="0"
+      class="irv-poll-option"
+      data-poll-option-id={{@option.id}}
+      data-poll-option-rank={{@option.rank}}
+    >
+      {{#if this.currentUser}}
+        <PollOptionIrvDropdown
+          @rank={{@option.rank}}
+          @option={{@option}}
+          @irvDropdownContent={{@irvDropdownContent}}
+          @sendRank={{this.sendRank}}
+        />
+      {{else}}
+        <DButton
+          class="btn btn-default"
```

Comment:
btwn is already set on DButton by default

Reply:
resolved https://github.com/discourse/discourse/pull/27155/commits/0674a866efd5f862d6f264ecb57718e7c7a8f973